source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+|         |            |     |       |          |          | count.  This     |
+|         |            |     |       |          |          | partition is     |
+|         |            |     |       |          |          | only enabled     |
+|         |            |     |       |          |          | after a system   |
+|         |            |     |       |          |          | maintenance.     |
+|         |            |     |       |          |          | Please alert the |
+|         |            |     |       |          |          | HD if you need   |
+|         |            |     |       |          |          | to runa job in   |
+|         |            |     |       |          |          | this partition.  |
++---------+------------+-----+-------+----------+----------+------------------+
+| ES      | eslogin_c5 | 1   | *max* | 12:00:00 | 16:00:00 | These jobs will  |
+|         |            |     |       |          |          | run on the C5    |
+|         |            |     |       |          |          | login nodes.     |
++         +------------+-----+-------+----------+----------+------------------+
+|         | eslogin_c6 | 1   | *max* | 12:00:00 | 16:00:00 | These jobs will  |
+|         |            |     |       |          |          | run on the C5    |
+|         |            |     |       |          |          | login nodes.     |
++         +------------+-----+-------+----------+----------+------------------+
+|         | dtn_f5_f6  | 513 | *max* | 12:00:00 | 16:00:00 | These jobs will  |
+|         |            |     |       |          |          | run on the DTN   |
+|         |            |     |       |          |          | nodes.  The DTN  |
+|         |            |     |       |          |          | nodes have both  |
+|         |            |     |       |          |          | F5 and F6        |
+|         |            |     |       |          |          | mounted.         |
++         +------------+-----+-------+----------+----------+------------------+
+|         | cron_c5    | 1   | *max* | 12:00:00 | 16:00:00 | Required         |
+|         |            |     |       |          |          | partition for    |
+|         |            |     |       |          |          | jobs run under   |
+|         |            |     |       |          |          | scron on the C5  |
+|         |            |     |       |          |          | login nodes.     |
++         +------------+-----+-------+----------+----------+------------------+
+|         | cron_c6    | 1   | *max* | 12:00:00 | 16:00:00 | Required         |
+|         |            |     |       |          |          | partition for    |
+|         |            |     |       |          |          | jobs run under   |
+|         |            |     |       |          |          | scron on the C6  |
+|         |            |     |       |          |          | login nodes.     |
++---------+------------+-----+-------+----------+----------+------------------+
```

Comment:
Lines 4-5 of the table both say "These jobs will run on the C5 login nodes."
I think the line for eslogin_c6 was meant to say C6?

Reply:
Fixed in a new push.